scout/commands/update/institute.py

```diff
@@ -29,7 +28,12 @@
     help="Specify email adress for a existing user that should be added to the institute",
 )
 @click.option("-d", "--display-name", help="Set a new display name for a insitute")
-@click.option("-l", "--loqusdb_id", help="Set a new loqusdb_id for a institute")
+@click.option(
```

Comment:
Loqusdb instances could be set from cli. I've changed the option to be multiple so you can update the institute wity more than one instance also from here. Tested locally and works fine